cypress/e2e/functional/student_registers_spec.js

```diff
@@ -42,7 +42,8 @@ context("Student registration tests", () => {
   });
 
   it("Register student - Error Scenarios", () => {
-    cy.get(signupPageElements.BTN_REGISTER_USER).click();
+    // use force:true because element can be hidden by the "Signed out Successfully" modal
+    cy.get(signupPageElements.BTN_REGISTER_USER).click({force: true}); 
```

Comment:
Would it make sense to have the test dismiss that "Signed out successfully" message first so we don't have to use `force: true`?